etc_fai/grub_base.cfg

```diff
@@ -0,0 +1,53 @@
+# make sure we can access partitions
+insmod part_msdos
+insmod part_gpt
+insmod all_video
+insmod gfxterm
+
+set gfxmode=auto
+set color_normal=white/black
+set color_highlight=red/black
+set menu_color_normal=white/black
+set menu_color_highlight=black/yellow
+
+
+menuentry "SEAPATH INSTALLER      |     _VERSIONSTRING_     |" --unrestricted { set timeout=20 }
+## BEGIN CUSTOM MENU ITEMS
+set default="  SEAPATH installation - Cluster mode with LVMRAID, English, no kerberos, no debug tools"
+set timeout=5
+menuentry "  SEAPATH installation - Cluster mode with LVMRAID, English, no kerberos, no debug tools" {
+    search --set=root --file /FAI-CD
+    linux   /boot/vmlinuz FAI_FLAGS="cluster,raid,verbose,sshd,createvt,reboot" FAI_ACTION=install FAI_CONFIG_SRC=file:///var/lib/fai/config rd.live.image root=live:CDLABEL=FAI_CD ipv6.disable=1 console=tty1
+    initrd  /boot/initrd.img
+}
+## END CUSTOM MENU ITEMS
+
+menuentry "Boot OS from first partition on first disk" {
+   if [ "$grub_platform" = "efi" ]; then
+      if [ -f (hd0,gpt1)/efi/debian/grub.cfg ]; then
+           configfile (hd0,gpt1)/efi/debian/grub.cfg
+      elif [ -f (hd1,gpt1)/efi/debian/grub.cfg ]; then
+           configfile (hd1,gpt1)/efi/debian/grub.cfg
+      elif [ -f (hd0,gpt2)/boot/grub/grub.cfg ]; then
+           configfile (hd0,gpt2)/boot/grub/grub.cfg
+      elif [ -f (hd1,gpt2)/boot/grub/grub.cfg ]; then
+           configfile (hd1,gpt2)/boot/grub/grub.cfg
+      else
+	   echo "cannot find grub.cfg"
+	   sleep 7
+      fi
+   # legacy BIOS booting
```

Comment:
Do we really want to support bios ? Or is it mandatory to have this part for grub.
IMO SEAPATH only support UEFI

Reply:
you are right, this is from the FAI default config, but we don't have to keep this since we are advocating that seapath only supports UEFI.
will remove.